pip/commands/help.py

```diff
@@ -31,6 +31,6 @@ def run(self, options, args):
             raise CommandError(' - '.join(msg))
 
         command = commands_dict[cmd_name]()
-        command.parser.print_help()
+        command.print_help()
```

Comment:
Maybe pass verbose=True here? or even options.verbose?

Reply:
Yes. but I forgot how to run pip from checkout. Rebased here - https://github.com/pypa/pip/pull/5746